examples/exex-hello-world/src/main.rs

```diff
@@ -42,6 +45,18 @@ async fn my_exex<Node: FullNodeComponents>(mut ctx: ExExContext<Node>) -> eyre::
     Ok(())
 }
 
+async fn ethapi_exex<Node>(
+    mut _ctx: ExExContext<Node>,
+    ethapi_rx: oneshot::Receiver<Arc<EthApiFor<Node>>>,
+) -> eyre::Result<()>
+where
+    Node: FullNodeComponents,
+{
+    let ethapi = ethapi_rx.await?;
+    info!("Received ethapi inside exex: {:?}", ethapi);
+    Ok(())
+}
```

Comment:
this is great, cool that this works,

since this is an example we should document all the steps here and describe what this actually does